File: noita-simulation/classes/solids/dynamic/sand.py
from classes.solids.dynamic.dynamicSolid import DynamicSolid


class Sand(DynamicSolid):
    def __init__(self, i, j) -> None:
        super().__init__(i, j)
        self.color = (255, 159, 25)

    def update(self, tiles: list[list]) -> list[list]:
        return super().update(tiles)

    

    

File: Tellus-cellular-automata/app/src/main/java/Blocks/Solids/DynamicSolid/Sand.java
package Blocks.Solids.DynamicSolid;

public class Sand extends DynamicParticle {
    
    public Sand() {
        super();

        // make them shift a bit to create texture
        // original sand = 182, 155, 99
        int offset = getColorOffset();
        setColors(132 + offset, 105 + offset, 50 + offset);

        setMaxSpeed(5);
        setAcceleration(0.6f);

        setBehaviours(true, true, true);
    }
}
